docs/root/version_history/current.rst

```diff
@@ -29,11 +29,12 @@ Minor Behavior Changes
   ``envoy.reloadable_features.send_strict_1xx_and_204_response_headers``
   (do not send 1xx or 204 responses with these headers). Both are true by default.
 * http: serve HEAD requests from cache.
+* http: the behavior of the present_match in route header matcher changed. The value of present_match is ignored in the past. The new behavior is present match performed when value is true. absent match performed when the value is false. Please reference :ref:`present_match
```

Comment:
Nit: `*present_match*` etc. (or RST links everywhere)

Reply:
got it, thanks